feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/usecases/steps/BuildStepsUseCase.kt

```diff
@@ -103,23 +82,126 @@ internal class BuildStepsUseCase @Inject constructor(
         )
     }.flatten()
 
-    private fun buildAgeSelectionStep(
+    fun buildCaptureAndMatchStepsForAgeGroup(
```

Comment:
I think this logic is complex enough that you can add unit tests for it to ensure the correct SDKs are selected

Reply:
Definitely! This has become the heart of SDK selection so I was already doing this.